app/components/Map/map_component.jsx

```diff
@@ -0,0 +1,233 @@
+                {address: "Oakland YMCA", mile: "1 mi", hour: "3 hrs"},
+                {address: "Tri-City Health Center", mile: "1.5 mi", hour: "2 hrs"}
+            ]
+        }
+
+        this.state = {
+
+            resource:false
+        }
+
+    //     this.resource = false;
+    //
+      this.updateResourceToTrue = this.updateResourceToTrue.bind(this)
+        this.updateResourceToFalse = this.updateResourceToFalse.bind(this)
+    }
+
+    updateResourceToTrue = () => {
+            this.setState({
+            resource:true});
+        }
+
+    updateResourceToFalse = () => {
+        this.setState({
+            resource:false});
+    }
+
+
+
+
+
+
+    render() {
+
+        const resource = this.state.resource;
+        const address = this.address.addr;
+        let header;
+
+        if(resource) {
+            header =
+            <div>resource is on</div>
+        } else {
+            header = <div><Grid container alignItems="center">
+
+                <DropDownMenuProps>
+                    <InputLabel>Distance </InputLabel>
+                    <Select
+                        labelId="distance"
+                        id="distance"
+                    >
+                        <MenuItem >0.5 miles</MenuItem>
+                        <MenuItem >1.5 miles</MenuItem>
+                        <MenuItem >2.5 miles</MenuItem>
+                    </Select>
+                </DropDownMenuProps>
+
+                <Divider orientation="vertical" flexItem />
+
+                <DropDownMenuProps>
+                    <InputLabel>Topic</InputLabel>
+                    <Select
+                        labelId="topics"
+                        id="topics"
+                    >
+                        <MenuItem > Mental Health </MenuItem>
+                        <MenuItem > LGBTQ </MenuItem>
+                        <MenuItem > BLM </MenuItem>
+                        <MenuItem > Food Banks </MenuItem>
+                        <MenuItem > Health Centers </MenuItem>
+                        <MenuItem > Community cleanup/art </MenuItem>
+
+
+                    </Select>
+                </DropDownMenuProps>
+
+
+            </Grid>
+
+            {(address).map((address, )=> (
+                <Card>
+                    <Card.Title title="Card Title" subtitle="Card Subtitle" left={LeftContent} />
+                    <Card.Content >
+                        <Title class = "card">
+                            {address.address}
+                            {"      "}
+                            {"Hours"}
+                        </Title>
+                        <Paragraph>
+                            {address.mile}
+                            {"      "}
+                            {address.hour}
+                        </Paragraph>
+                    </Card.Content>
+                    {/*<Card.Cover source={{ uri: 'https://picsum.photos/700' }} />*/}
```

Comment:
Maybe a comment to explain the new changes would help. What did 119 do differently and why did it get removed (it seems like a pretty cool concept to have random images :))).